libraries/botframework-streaming/src/disassemblers/requestDisassembler.ts

```diff
@@ -5,38 +5,38 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
+
+import { IRequestPayload, IStreamWrapper } from '../interfaces';
 import { PayloadTypes } from '../payloads/payloadTypes';
-import { PayloadSender } from '../payloadTransport/payloadSender';
-import { StreamingRequest } from '../streamingRequest';
+import type { PayloadSender } from '../payloadTransport/payloadSender';
+import type { StreamingRequest } from '../streamingRequest';
 import { PayloadDisassembler } from './payloadDisassembler';
-import { IStreamWrapper } from '../interfaces/IStreamWrapper';
-import { IRequestPayload } from '../interfaces/IRequestPayload';
 
 /**
  * Streaming request disassembler.
  */
 export class RequestDisassembler extends PayloadDisassembler {
-    public request: StreamingRequest;
-    public payloadType: PayloadTypes = PayloadTypes.request;
+    payloadType: PayloadTypes = PayloadTypes.request;
 
     /**
      * Initializes a new instance of the [RequestDisassembler](xref:botframework-streaming.RequestDisassembler) class.
+     *
      * @param sender The [PayloadSender](xref:botframework-streaming.PayloadSender) to send the disassembled data to.
      * @param id The ID of this disassembler.
      * @param request The request to be disassembled.
      */
-    public constructor(sender: PayloadSender, id: string, request?: StreamingRequest) {
+    constructor(sender: PayloadSender, id: string, public request?: StreamingRequest) {
         super(sender, id);
-        this.request = request;
     }
 
     /**
      * Gets the stream this disassembler is operating on.
+     *
      * @returns An [IStreamWrapper](xref:botframework-streaming.IStreamWrapper) with a Subscribable Stream.
      */
-    public async getStream(): Promise<IStreamWrapper> {
-        const payload: IRequestPayload = { verb: this.request.verb, path: this.request.path, streams: [] };
-        if (this.request.streams) {
+    async getStream(): Promise<IStreamWrapper> {
+        const payload: IRequestPayload = { verb: this.request?.verb, path: this.request?.path, streams: [] };
+        if (this.request?.streams) {

```

Comment:
You _might_ be able to simplify further as `this.request?.streams?.forEach(...)` without the surrounding `if`.